pom.xml

```diff
@@ -13,17 +14,20 @@
   <packaging>maven-plugin</packaging>
 
   <name>maven-license-plugin Maven Plugin</name>
-  <url>http://maven.apache.org</url>
+  <url>https://maven.apache.org</url>
```

Comment:
???

Reply:
well one step better than before =/ this plugin has no docs